_includes/2022-03-25-emergent-symbols/identity_rules.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head><meta charset="utf-8" /></head>
+<body>
+  <figure>
+    <img style="margin: 0 auto 1em auto; display: block;" width="400"
+       src="{{ site.url }}/public/images/2021-12-01-emergent-symbols/identity_rules.png"
```

Comment:
In all of the HTML figures that you have in `_includes/`, you'll need to update the path to point to `2022-03-25-emergent-symbols` (sorry about that)

Reply:
Done!